server/datastore/mysql/migrations/tables/20240322145615_CreateTableNanoDDMRequests_test.go

```diff
@@ -0,0 +1,20 @@
+package tables
+
+import "testing"
+
+func TestUp_20240322145615(t *testing.T) {
```

Comment:
Can probably safely delete this file, not much tests make sense for a table creation migration.

Reply:
Removed